code/modules/client/preferences.dm

```diff
@@ -139,7 +139,7 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 	var/see_rc_emotes = TRUE
 
 	///Should we automatically fit the viewport?
-	var/auto_fit_viewport = TRUE
+	var/auto_fit_viewport = FALSE
```

Comment:
No

Reply:
It's a /tg/ port.

Reply:
It is not

Reply:
I would not even try to make the change if /tg/ was not already that way. They seem to have no problem with the deafult of `FALSE` https://github.com/tgstation/tgstation/blob/master/code/modules/client/preferences.dm#L120

Reply:
It would be super cool if you would checkout and run this locally so you can see how if your viewport is already "fit" it'll stay that way when you connect as a guest and I am betting you'd never even notice this change had this PR not triggered you.